src/cpu/x64/jit_uni_pool_kernel.cpp

```diff
@@ -132,8 +132,14 @@ jit_uni_pool_kernel_t<isa>::jit_uni_pool_kernel_t(
     if (jpp.ind_dt == data_type::s32) dtypes.insert(data_type::f32);
     if (jpp.needs_f32_accum_for_bf16) dtypes.insert(data_type::f32);
 
+    typename io_mdt_helper::saturation_map_t saturation_confs;
+    if (dtypes.count(data_type::u8)) {
+        saturation_confs[data_type::u8]
+                = io::io_saturation_conf_t(0, 1, tmp_gpr);
```

Comment:
`0` and `1` must be a named constants and play together with other vector registers in this implementation's picture. This way it's very sneaky and hard to find any potential register collision.